### PR TITLE
User story 35 36

### PR DESCRIPTION
**Pull request by tschaffer1618**
-Add functionality for merchant admin to activate/deactivate items

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -38,4 +38,8 @@ def cannot_fulfill_message
       "Cannot fulfill. There are no #{self.name} items remaining."
     end
   end
+
+  def toggle_status
+    self.toggle!(:active?)
```
Comment (nancylee713) [inline app/models/item.rb]: Nice use of active record method! This is so neat

File: spec/features/users/merchant_users/add_item_spec.rb [hunk 2]
```diff
@@ -51,7 +51,7 @@
             expect(page).to have_content("$#{price}")
             expect(page).to have_css("img[src*='#{image_url}']")
             expect(page).to have_content(inventory)
-            expect(page).to have_content("true")  # how to check contents for enabled? and active? they need to be more specfic
+            expect(page).to have_content("Active")  # how to check contents for enabled? and active? they need to be more specfic
```
Comment (nancylee713) [inline spec/features/users/merchant_users/add_item_spec.rb]: Thanks for updating the contents 👍

**Comment by nancylee713** [comment]
The activate/deactivate buttons work so seamlessly! I was just wondering if we need another test on `items_path`, to double check that a deactivated item is no longer showing up on the (regular) items index page?

File: app/controllers/merchant/items_controller.rb [hunk 3]
```diff
@@ -45,6 +45,18 @@ def update
...
+    @item = Item.find(params[:item_id])
+    @item.toggle_status
+    # @merchant.items.find(@item.id).toggle_status
+    if @item.active?
+      flash[:success] = "This item is now available for sale"
+    else
+      flash[:success] = "This item is no longer for sale"
+    end
+    redirect_to merchant_user_index_path
+  end
```
Comment (leiyakenney) [inline app/controllers/merchant/items_controller.rb]: Can we condense this if/else into one line?

**Pull request merged**